### PR TITLE
Finishing touches towards 0.1.0

### PR DESCRIPTION
**Pull request by connectbo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,35 +2,34 @@
...
 A web application for managing an iRODS Zone.
 
-## Installation & Run
+## Setup
 
-### 1. Clone and repository from Github
+### 1. Install Docker and Docker Compose
+Follow instructions in Docker docs to install Docker and Docker Compose for your OS.
+
+### 2. Clone and repository from Github
```
Comment (trel) [inline README.md:10]: "Clone and" -> "Clone the"
"Github" -> "GitHub"

File: README.md [hunk 2]
```diff
@@ -2,35 +2,34 @@
...
 Clone the repository from Github.
```
Comment (trel) [inline README.md:11]: "Github" -> "GitHub"

or... remove this line, as it's the same as the section header

**Comment by connectbo** [comment]
Yup, added. Trying to think if we are missing anything here.

**Comment by trel** [comment]
maybe a screen shot near the top of the readme, showing... a treeview?

i think that may be it...

**Comment by trel** [comment]
very nice... however...  this highlights the fact that the treeview doesn't show what TYPE of resource each one is...

like...
```
$ ilsresc
pt1:passthru
└── ufs3:unixfilesystem
    └── testResc15:unixfilesystem
        ├── tescResc9:replication
        │   └── pt2:passthru
        │       ├── testResc13:unixfilesystem
        │       └── ufs2:unixfilesystem
        │           ├── testResc2:unixfilesystem
        │           │   ├── testResc18:passthru
        │           │   └── testResc5:unixfilesystem
        │           └── testResc3:unixfilesystem
        └── testResc:unixfilesystem
            └── testResc14:unixfilesystem
random1:random
├── tescRsec7:unixfilesystem
│   └── testResc10:unixfilesystem
│       └── testResc11:replication
└── testResc12:univmss
    └── testResc4:unixfilesystem
testResc6:structfile
└── mockarchive1:mockarchive
    └── demoResc:unixfilesystem
        ├── testResc8:unixfilesystem
        └── testingResc:unixfilesystem
            ├── ufs1:unixfilesystem
            └── univmss1:univmss
```

can you make it do that?   and then retake the screenshot?  :P

**Comment by connectbo** [comment]
ahhh, along with its type.. I see, will make it do the job

**Comment by trel** [comment]
made #90 for this

**Comment by connectbo** [comment]
Alright, each resource will now display its type in parenthesis along side the name. Refactored some logic when changing hierarchy, and work as expected.

File: src/components/draggable-tree/tree.js [hunk 3]
```diff
@@ -166,23 +166,24 @@ export const Tree = (props) => {
     }
 
     const renderTreeNode = (node) => {
+        const nodeName = `${node[0]}${node[0] !== 'tempZone' ? `(${node[1]})` : ''}`
```
Comment (trel) [inline src/components/draggable-tree/tree.js:169]: i'm concerned about the hard-coded 'tempZone'... is that... good?

Comment (connectbo) [reply src/components/draggable-tree/tree.js:169]: No, it's not good. It should come from the zone report. Forgot that :)

**Comment by trel** [comment]
definitely need a space between the name and the `()`.

**Comment by connectbo** [comment]
Gotcha, it's updated. Let's see how it looks like.

**Comment by trel** [comment]
yes, the space works.     not convinced it's better than a single colon... but that could just be what i'm used to seeing on the command line...

**Comment by trel** [comment]
I think this is good enough to get merged for now - please # em.

File: src/components/draggable-tree/tree.js (hunk 3 above)
Comment (trel) [reply src/components/draggable-tree/tree.js]: sold.

**Comment by connectbo** [comment]
Actually I felt the same - the colon one feels cleaner than this (plus it's consistent with the cmd line). Happy to change back to colon, your call.

**Comment by trel** [comment]
i'll change to colon here... one sec...

**Comment by trel** [comment]
colon is better.

**Comment by connectbo** [comment]
cool, I will change that and update the screenshot.

**Pull request merged**